src/build/Calculator.StampAssemblyInfo.targets

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+    For .NET Core SDK-style projects, the "Version" MSBuild property is written into assembly files
+    automatically. This target provides the same feature in non-SDK-style projects using a
+    generated file.
+  -->
+
+  <PropertyGroup Condition="'$(Version)' != ''">
```

Comment:
You'll probably want to use `$(AppVersion)` instead of `$(Version)` in this file, since AppVersion is what's being passed on the command line [here](https://github.com/microsoft/calculator/blob/233339d289ca1f422b3371e99a20311f93625ade/build/pipelines/templates/build-single-architecture.yaml#L32).

Another option would be to change build-single-architecture.yaml to use `Version` instead of `AppVersion`. If you do that you'll need to change other projects in the repo which currently use `AppVersion`.

Reply:
Thanks Matt,
I decided to use `Version` instead of `AppVersion` since build-single-architecture.yaml is the only producer who specifies `AppVersion`. Please see commit [43634b9](https://github.com/microsoft/calculator/pull/1598/commits/43634b9c368d038ed4112ab7d935889a1a825e81) for details.

Also, edited AssemblyInfo.cs to provide the information that was declared in Calculator.rc. Changes are in commit [
4ed33a7](https://github.com/microsoft/calculator/pull/1598/commits/4ed33a7d405edf314889bcd25c5be62ae482d527).

After CI-pipeline run finished, I checked the properties of file CalculatorApp.exe and it has the expected information.
![image](https://user-images.githubusercontent.com/60599517/125058291-b63fe600-e0dc-11eb-8593-855a8a0e4581.png)

Reply:
Sounds good--just make sure that the .dll files in the package still have version information just like CalculatorApp.exe.

Reply:
Double checked the master branch, only Calculator project(the UI project) has assembly-info. The other projects, such as ***CalcViewModel***, ***GraphControl***, ***GraphingImpl***, ***TraceLogging***, they are tagging with default assembly-info which is empty. ***CalcViewModel*** was a static-library, so it's fine if it doesn't have assembly-info, but the remaining projects don't look good.

I've assigned some assembly-info for all the shared library(.dll) projects in commit [8926e99](https://github.com/microsoft/calculator/pull/1598/commits/8926e99d7efbaf22557c4065534cb91fcc2f5b72). Let me know if it's not good to do so and I'll revert the commit.

BTW, if we want to add assembly-info for ***GraphingImpl.dll***, we need to submit some changes in the internal repo. But that task should not block this PR from getting checked in, I suppose.

Reply:
Hi @mcooley , this PR has been paused for a while. We'd like to make it checked in first. If there's any problems, we can send out other PRs to fix them.